File: examples/layouts/layouts.nim
import fidget2, bumpy

use("https://www.figma.com/file/fn8PBPmPXOHAVn3N5TSTGs")

onDisplay "/Layout1":
  thisNode.box.wh = windowSize

startFidget(
  windowTitle = "Layouts",
  entryFrame = "Layout1",
  resizable = true
)

File: examples/javascript/layouts.js
var ffi = require('ffi');

var fidget = ffi.Library('libfidget', {
    'startFidget': [ 'void', [ 'string', 'string', 'string', 'bool' ] ]
});

fidget.startFidget(
    "https://www.figma.com/file/fn8PBPmPXOHAVn3N5TSTGs",
    "JavaScript Layouts",
    "Layout1",
    true
)
